### PR TITLE
#DX Improvements ❤️ ✨

### PR DESCRIPTION
**Pull request by jameshopkins**
* Add `yarn.lock` file
* Remove fully qualified paths in `package.json`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +37,7 @@
   },
   "homepage": "https://github.com/craftship/yith#readme",
   "dependencies": {
+    "aws-sdk": "^2.7.10",
```
Comment (jonsharratt) [inline package.json]: Just a heads up, the reason this is not in `dependencies` is that the Lambda execution environment already includes this by default.  Only required locally when serving the functions via the Serverless webpack plugin.

Reference: http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html

Comment (jameshopkins) [reply package.json]: Ah I see. The issue I see with this approach, is that there's not a single source of truth for *all* your dependencies - i.e you're dependent on a globally-defined dependency in the Lambda runtime.

You _may_ find that this dependency gets bumped independently of what is (currently) defined in your `devDependencies`, leading to a disparity between local and Lambda environments.

Comment (jonsharratt) [reply package.json]: Yer my plan was to just keep an eye on it (it should actually be in dev-dependecies and locked to `2.7.10` not `^2.7.10`, currently the bundle is ~1.7mb or so.  Low risk as from this point forward I doubt they will change it, they are pretty slow on updating things to be fair from past experience.

**Comment by jonsharratt** [comment]
Sorry boss, just realised I merged in a change that updates the `serverless-webpack` dev dependency meaning the `yarn.lock` files is out of date now, could you merge in the latest master to your branch and run `yarn` to update the lock file.

Also as discussed earlier, in the `devDependencies` we should probably lock it down to the current AWS version explicitly `2.7.10` instead of `^2.7.10`.

👍

**Comment by jameshopkins** [comment]
> Also as discussed earlier, in the devDependencies we should probably lock it down to the current AWS version explicitly 2.7.10 instead of ^2.7.10.

Yarn guarantees the determinism of dependencies when doing a `yarn install` - regardless of whether a caret prepends the version number of not. The caret only limits the scope of a dependency update when using `yarn update`.

**Comment by jonsharratt** [comment]
Tested locally and `yarn.lock` file all up to date, thanks for this, much appreciated 👍

**Pull request merged**